### DISCUSSION TITLE
Letsencrypt not accessible - EACCES (Bug report)

### DISCUSSION THREAD

**Discussion by Pupo1** [TriliumNext/Trilium#3419] [General]
### Trilium Version

0.57.5

### What operating system are you using?

Other Linux

### What is your setup?

Server access only

### Operating System Version

Debian 11

### Description

I tried the TLS configuration described [here](https://github.com/zadam/trilium/wiki/TLS-configuration) to work together with my letsencrypt certificates.

But I receive the error:
`error: EACCES: permission denied, open '/ets/letsencrypt/live/(my domain)/privkey.pem`

Which is understandable, because the privkey.pem is only readable by root, while I start the service as a normal user. 

### Error logs

13:54:25.819 Table counts: notes: 173, note_revisions: 0, branches: 191, attributes: 247, etapi_tokens: 0                                                                                                          
13:54:25.830 All consistency checks passed with no errors detected (took 12ms)                                                                                                                                     
13:57:24.922 DB size: 1634 KB                                                                                                                                                                                      
13:57:26.122 Trusted reverse proxy: false                                                                                                                                                                          
13:57:26.153 Becca (note cache) load took 11ms                                                                                                                                                                     
13:57:26.161 Error: EACCES: permission denied, open '/etc/letsencrypt/live/( my domain)/privkey.pem'

**Comment by zadam** [comment]
Well, then you can either make the private key accessible to the trilium process (run trilium as root or chmod cert to user) or e.g. run a TLS terminating proxy like nginx in front of the trilium process (that's what I do).

**Reply by Pupo1** [reply]
Okay, so I made a reverse proxy for the trilium process, and it works, thank you. I wonder if it shouldn't be mentioned in the installation docs.

**Reply by zadam** [reply]
I mentioned the option of reverse proxy into the https://github.com/zadam/trilium/wiki/TLS-configuration

**Comment by Pupo1** [comment]
Okay, but in the section "Modifying config.ini" on the same page, you did not mention that the certificates' rights must be changed or trilium won't be able to access them.
